src/StreamJsonRpc/Reflection/MessageFormatterRpcMarshaledContextTracker.cs

```diff
@@ -295,6 +334,9 @@ public MarshalToken(int __jsonrpc_marshaled, long handle, string? lifetime)
 
         [DataMember(Name = "lifetime", EmitDefaultValue = false)]
         public string? Lifetime { get; }
+
+        [DataMember(Name = "subTypeCode", EmitDefaultValue = false)]
```

Comment:
This addendum to the protocol should be reflected in [the doc](https://github.com/microsoft/vs-streamjsonrpc/blob/main/doc/general_marshaled_objects.md) for it as well.

Reply:
and a nit: call it `subType` instead of `subTypeCode`. `Code` doesn't really add any value, and could be added as a suffix to almost anything. This is feedback on the wire protocol property name specifically. The C# property name of `SubTypeCode` is totally fine, as it makes it clear that the type is an integer rather than a `Type`.